tools/Status/Status.py

```diff
@@ -340,6 +349,15 @@ def render_status(show_paths, show_unidentified, **kwargs):
               default=None,
               help=("On a new screen, refresh jobs every 'watch' number "
                     "of seconds. Exit out with Ctl+C."))
+@click.option("--state-styles",
+              "input_state_styles",
+              type=dict,
+              default=None,
```

Comment:
If you make this an empty dict then it's easier to handle (`dict.update` does nothing).